composer.json

```diff
@@ -47,7 +47,8 @@
     },
     "autoload": {
         "classmap": [
-            "Commands/"
+            "Commands/",
+            "src/"
```

Comment:
So I'm not sure we should be using the `src` directory to hold ComposerConstraint, since that's automatically mapped to the `Drupal\lightning_dev` namespace by Drupal core itself. So instead, I'd say we should just put ComposerConstraint.php in the root directory, and add it to the classmap as 'ComposerConstraint.php'. We can also remove the Drupal\lightning_dev namespace from the class.

Normally this wouldn't be an issue, but autoloading can be very finicky, and if we do this wrong, a lot of other  things can break across all components.

Another possibility is to not use the autoloader at all, and just do `require __DIR__ . '/ComposerConstraint.php` at the top of RoboFile.php, since that's the only place this class is used.

Reply:
It's also used in the test

Reply:
You are, of course, correct. I'm _especially_ dumb on Mondays.

Let's go with a modified version of the first idea, then -- move the class file to the Commands directory and into the Acquia\Lightning\Commands namespace, since we know that namespace already works and doesn't interfere with anything.